be/src/category/dto/update-category.input.ts

```diff
@@ -10,4 +11,7 @@ export class UpdateCategoryInput {
     nullable: true,
   })
   bundelable?: boolean;
+
+  @Field(type => [GraphQLInt], { nullable: true })
+  products: number[] | null;
```

Comment:
I think it would be better to call this filed `productIds`. (Also other ones like this as well)